.travis.yml

```diff
@@ -14,10 +14,13 @@ matrix:
       chrome: stable
     before_script:
       - (test -x $HOME/.cargo/bin/cargo-install-update || cargo install cargo-update)
-      - (test -x $HOME/.cargo/bin/cargo-generate || cargo install --vers "^0.2" cargo-generate)
+      - (test -x $HOME/.cargo/bin/cargo-generate || cargo install --vers "0.2.2" cargo-generate)
```

Comment:
If we get a cached `cargo-generate`, and therefore don't execute the updated `cargo install` command here, whill `cargo install-update -a` bump minor versions of an installed tool? If not, we may need to do two pushes: one that doesn't use any chached tool to force an install of 0.2.2, and then another to re-enable the use of cached tools. I think I've been bitten by this before. Not sure what the best way to deal with this is going forward.